apps-rendering/src/components/NewsletterSignup/EmailSignupForm.tsx

```diff
@@ -0,0 +1,77 @@
+// ----- Imports ----- //
+
+import { css } from '@emotion/react';
+import { neutral, space, textSans } from '@guardian/source-foundations';
+import { Button, Label, TextInput } from '@guardian/source-react-components';
+import type { NewsletterSignUp } from 'bodyElement';
+import type { FC } from 'react';
+
+// ----- Component ----- //
+
+interface Props {
+	newsletter: NewsletterSignUp['newsletter'];
+}
+
+const formStyle = css`
+	display: flex;
+	align-items: center;
+	flex-wrap: wrap;
+`;
+
+/**
+ * NOTE: this component is non functional and is for demonstration only.
+ * The UI for the NewsletterSignup might not use an HTML form for apps
+ * when implemented
+ */
+const EmailSignupForm: FC<Props> = ({ newsletter }) => {
+	const { id } = newsletter;
+
+	const handleSubmit = (): void => {
+		console.log({ id });
+	};
+
+	return (
+		<>
+			<Label
+				text="Enter your email address"
+				cssOverrides={css`
+					div {
+						${textSans.xsmall({ fontWeight: 'bold' })};
+					}
+				`}
+			/>
+			<form css={formStyle} action={undefined}>
+				<TextInput
+					type={'email'}
+					width={30}
+					hideLabel
+					label={'Enter your email address'}
+					cssOverrides={css`
+						height: 36px;
+						margin-right: ${space[3]}px;
+						margin-top: 0;
+						margin-bottom: ${space[2]}px;
+						flex-basis: 335px;
```

Comment:
We use `rem`s in AR to make sure the UI scales evenly when the user increases their zoom level. I've demonstrated a few different ways to get `rem` values here:

- `remSpace` from Source, this is equivalent to `space`
- Write the `rem` value directly instead of the `px` value
- Use the `pxToRem` helper from Source

```suggestion
						height: 2.25rem;
						margin-right: ${remSpace[3]};
						margin-top: 0;
						margin-bottom: ${remSpace[2]};
						flex-basis: ${pxToRem(335)}rem;
```